cms/envs/devstack_optimized.py

```diff
@@ -0,0 +1,51 @@
+"""
+Settings to run Studio in devstack using optimized static assets.
+
+This configuration changes Studio to use the optimized static assets generated for testing,
+rather than picking up the files directly from the source tree.
+
+The following Paver command can be used to run Studio in optimized mode:
+
+  paver devstack studio --optimized
+
+You can also generate the assets explicitly and then run Studio:
+
+  paver update_assets cms --settings=test_static_optimized
+  paver devstack studio --settings=devstack_optimized --fast
+
+Note that changes to JavaScript assets will not be picked up automatically
+as they are for devstack. Instead, update_assets must be invoked each time
+that changes have been made.
+"""
+
+# We intentionally define lots of variables that aren't used, and
+# want to import all variables from base settings files
+# pylint: disable=wildcard-import, unused-wildcard-import
+
+# Pylint gets confused by path.py instances, which report themselves as class
```

Comment:
I would get rid of this comment block too and just put the pylint disable statement on line 30 (pylint: disable=no-name-in-module).